### ISSUE TITLE
ValueError: Unpack failed: incomplete input

### ISSUE THREAD

**Issue by dejianguy** [explosion/spaCy#3486]
## Background
I was trying to install Spacy on one of the environment that I created on Anaconda. The commands that i used to install are as follows 

First i created a new environment using Conda and called it workingspacy and i activated the environment.
**conda create -n workingspacy --clone base
conda activate workingspacy**

I then installed spacy on this environment

**conda install -c conda-forge spacy
python -m spacy download en**

While trying to run the code, i am faced with this error... I think it has something to do with the msgpack or Srsly package. Can anyone offer some advice on this?

The following below is my error code
```bash
runfile('C:/Users/Oei/Desktop/MaskingProject/TrainingEntity.py', wdir='C:/Users/Oei/Desktop/MaskingProject')
Traceback (most recent call last):

  File "<ipython-input-2-b2ab34f52ac7>", line 1, in <module>
    runfile('C:/Users/Oei/Desktop/MaskingProject/TrainingEntity.py', wdir='C:/Users/Oei/Desktop/MaskingProject')

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 704, in runfile
    execfile(filename, namespace)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 108, in execfile
    exec(compile(f.read(), filename, 'exec'), namespace)

  File "C:/Users/Oei/Desktop/MaskingProject/TrainingEntity.py", line 129, in <module>
    plac.call(main)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\plac_core.py", line 328, in call
    cmd, result = parser.consume(arglist)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\plac_core.py", line 207, in consume
    return cmd, self.func(*(args + varargs + extraopts), **kwargs)

  File "C:/Users/Oei/Desktop/MaskingProject/TrainingEntity.py", line 64, in main
    nlp = spacy.load(model)  # load existing spaCy model

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spacy\__init__.py", line 27, in load
    return util.load_model(name, **overrides)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spacy\util.py", line 133, in load_model
    return load_model_from_path(Path(name), **overrides)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spacy\util.py", line 173, in load_model_from_path
    return nlp.from_disk(model_path)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spacy\language.py", line 786, in from_disk
    util.from_disk(path, deserializers, exclude)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spacy\util.py", line 611, in from_disk
    reader(path / key)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spacy\language.py", line 776, in <lambda>
    deserializers["tokenizer"] = lambda p: self.tokenizer.from_disk(p, exclude=["vocab"])

  File "tokenizer.pyx", line 390, in spacy.tokenizer.Tokenizer.from_disk

  File "tokenizer.pyx", line 431, in spacy.tokenizer.Tokenizer.from_bytes

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\spacy\util.py", line 587, in from_bytes
    msg = srsly.msgpack_loads(bytes_data)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\srsly\_msgpack_api.py", line 29, in msgpack_loads
    msg = msgpack.loads(data, raw=False, use_list=use_list)

  File "C:\Users\Oei\Anaconda3\envs\workingspacy\lib\site-packages\srsly\msgpack\__init__.py", line 60, in unpackb
    return _unpackb(packed, **kwargs)

  File "_unpacker.pyx", line 203, in srsly.msgpack._unpacker.unpackb

ValueError: Unpack failed: incomplete input
```

## Your Environment
* Operating System: Windows 8.1 Enterprise
* Python Version Used: spyder 3.3.2 python 3.7
* spaCy Version Used: 2.1.3 . using en_core_web_sm
* Environment Information: Anaconda 3
* Msgpack Version Used: 0.5.6
* Srsly Version Used: 0.0.5

**Comment by honnibal** [comment]
Hmm! Never seen that before. Very puzzled about what it could be. Just for sanity, would you mind posting the checksum of the tokenizer file? I think this should work:

```
python -c  "import en_core_web_sm, pathlib, hashlib; tok_loc = pathlib.Path(en_core_web_sm.__file__).parent / 'en_core_web_sm-2.1.0' / 'tokenizer'; print(hashlib.md5(tok_loc.open('rb').read()).hexdigest())"
```

**Comment by dejianguy** [comment]
![image](https://user-images.githubusercontent.com/48939934/55045548-15e36600-5079-11e9-9182-8012f9b326d5.png)

Hi honnibal, thank you very much for getting back to me. I hope the above image is what you are looking for as the concept of checksum is still very new to me.

**Comment by ines** [comment]
@dejianguy Thanks – this looks correct. Basically, we were trying to find out whether the file included in the model on your machine is complete and correct. But I just ran the same command @honnibal posted on my machine and it returns the same result. This means that the file on my machine is the same as the file on your machine.

Have you tried uninstalling and reinstalling the model? You can run `pip uninstall en_core_web_sm` to uninstall it. Also make sure to check the `spacy/data` directory in your  `site-packages` and make sure there's nothing in there.

**Comment by dejianguy** [comment]
Hi guys, yea i just tried reinstalling and installing the model but it did not work.. Do i have to uninstall Spacy entirely too?

**Comment by ines** [comment]
Just released v2.2, which comes with new models. Could you try again with the new version and models? If the issue still persists, feel free to reopen.
